docs/user-guide/administration/backup-restoration.md

```diff
@@ -0,0 +1,158 @@
+# GreptimeDB Export & Import Tools
+
+This guide describes how to use GreptimeDB's Export and Import tools for database backup and restoration.
+
+## Table of Contents
+- [Overview](#overview)
+- [Export Tool](#export-tool)
+- [Import Tool](#import-tool)
+- [Common Usage Scenarios](#common-usage-scenarios)
+- [Best Practices](#best-practices)
+- [Troubleshooting](#troubleshooting)
+
+## Overview
```

Comment:
```suggestion
```

since we have the navigation inside the pages, so we do not need to write the table of contents manually:

<img width="359" alt="image" src="https://github.com/user-attachments/assets/49e6e5a5-ae15-4557-823c-e8d223dc96de">